docs/docs/uploads-legacy.md

```
@@ -0,0 +1,947 @@
+  urlSigner,
+})
+
+export { saveFiles, storagePrismaExtension }
+```
+
+**1. Upload Configuration** This is where you configure the fields that will
+receive uploads. In our case, it's the `profile.avatar` field.
+
+The shape of the config looks like this:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)